src/main/java/frc/robot/Robot.java

```diff
@@ -11,7 +11,9 @@
 import edu.wpi.first.wpilibj.TimedRobot;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
-import frc.robot.Library.Pneumatics.SubSys_Pneumatics;
+import frc.robot.library.pneumatics.SubSys_Pneumatics;
+
+import java.io.IOError;
```

Comment:
If it is unused then it may be a reminant from a yagsl implementation somehow which I belive throws an IO error if it fails to parse the json files for the swerve

Reply:
To be fixed at update to 2024